etc/calico/confd/templates/bird_aggr.cfg.template

```diff
@@ -1,22 +1,29 @@
 # Generated by confd
 # ------------- Static black hole addresses -------------
-{{if ls "/"}}
+{{- $block_key := printf "/calico/ipam/v2/host/%s/ipv4/block" (getenv "NODENAME")}}
+{{- $static_key := "/calico/staticroutes"}}
+{{if or (ls $block_key) (ls $static_key)}}
 protocol static {
-{{range ls "/"}}
+{{range ls $block_key}}
 {{$parts := split . "-"}}
 {{$cidr := join $parts "/"}}
    route {{$cidr}} blackhole;
 {{end}}
+{{- range ls $static_key}}
```

Comment:
Might be worth a comment here, so we can distinguish the static routes from the block routes in the rendered templates.